lib/jxl/decode.cc

```diff
@@ -2270,7 +2266,8 @@ JxlDecoderStatus JxlDecoderGetColorAsICCProfile(const JxlDecoder* dec,
   JxlDecoderStatus status =
       JxlDecoderGetICCProfileSize(dec, target, &wanted_size);
   if (status != JXL_DEC_SUCCESS) return status;
-  if (size < wanted_size) return JXL_API_ERROR("ICC profile output too small");
+  if (size < wanted_size)
+    return JXL_INPUT_ERROR("ICC profile output too small");
```

Comment:
This is actually an API error, we get here if the buffer provided in the API call was too small.

Reply:
right, changing it